lib/amazonka/test/Main.hs

```diff
@@ -0,0 +1,97 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
Please add the standard module header. Also, explicit extensions are not required because they'll be pulled in by `amazonka.cabal` file once you import its `base` stanza in the `tests` target.